client/src/components/UserForm.css

```diff
@@ -9,6 +9,10 @@
   margin-bottom: 25px;
 }
 
+.container-input {
+  width: 50%;
```

Comment:
There are some utility classes in bootstrap. I'd recommend using them wherever possible as they are more verbose and you can avoid writing / maintaining CSS.
Here the utility class is called `w-50`.
https://getbootstrap.com/docs/5.0/utilities/sizing/

Reply:
Hmm, I see you were using this before. What motivated the change?

Reply:
I didn't think about using it this way. I've made the change.